content/docs/how-we-work/coding-principles-standards.md

```diff
@@ -1,82 +1,68 @@
+  - When writing code, our code should solve ONLY and EXACTLY the use case that we are trying to support. In many instances, we have a desire to make things MORE FLEXIBLE in the FUTURE but that is code we will have to support without knowing whether any of those use cases will ever come up. Another big reason for wanting to be FLEXIBLE is that refactoring is hard, and it feels easier to have flexibility at the front, but that is why we write automated tests, so refactoring will not be as painful.
 
 ### Drupal-specific Coding Standards
 
-We follow [Drupal Coding Standards](https://www.drupal.org/docs/develop/standards) which includes PHP, CSS, JavaScript, Twig, YAML, and more.
+- We follow [Drupal Coding Standards](https://www.drupal.org/docs/develop/standards) which includes PHP, CSS, JavaScript, Twig, YAML, and more.
 
-- Also [Writing Secure Code in Drupal](https://www.drupal.org/docs/administering-a-drupal-site/security-in-drupal/writing-secure-code-for-drupal)
+- [Writing Secure Code in Drupal](https://www.drupal.org/docs/administering-a-drupal-site/security-in-drupal/writing-secure-code-for-drupal)
 
 - We ensure [Code Quality with DCQ](https://www.drupal.org/project/dcq) which includes the Coder module to ensure code meets Drupal standards.
 
 - We use namespaces so that we avoid namespace collisions.
 
-  - All module functions (even little helper functions) should have the module prefix
-  - Database tables should have the module prefix.
-  - Variables should have the module prefix.
-  - Custom code, when possible, should be developed as a standalone library that a module can include and integrate with Drupal. This should mean using class namespaces and best OOP practices, and at that point we can use an alternative more flexible autoloading approach, like the one provided by composer.
-
-- We always try to use Drupal functions where they exist. This helps with upgrades (among other things).
-
-  - Use `drupal_get_path();` to create a path to a file in your module directory.
-  - Use `path_to_theme();` to create a path to a file in the current theme directory.
+- We always try to use Drupal functions where they exist. This helps with upgrades, among other things.
 
-- We use the theme system! When theming, don't hack module code when it is possible to use a theme function instead.
-  -The corollary: when coding modules keep all markup in theme functions!
+- We use a theme system. When theming, don't hack module code when it is possible to use a theme function instead.
+  -The corollary: when coding modules, keep all markup in theme functions.
```

Comment:
The typo in bullet still exists. Look at this line in the preview (https://deploy-preview-102--axelerant-engg-handbook.netlify.app/docs/how-we-work/coding-principles-standards/) to see what's going wrong.